roles/postgresql/tasks/postgresql_yum7.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: yum | Initialize postgresql
+  become: yes
+  yum:
+    name: "{{ item }}"
+    state: installed
+    update_cache: yes
+  with_items:
+    - postgresql-server
+    - postgresql-contrib
+    - postgresql-devel
+  tags: [db, postgresql]
+
+- name: yum | Initialize Postgresql
+  become: yes
+  command: postgresql-setup initdb
+  tags: [db, postgresql]
+
+- name: yum | MD5-encrypted password for postgres 1
+  become: yes
+  command: 'sed -i "s/\(host.*all.*all.*127.0.0.1\/32.*\)ident/\1md5/" /var/lib/pgsql/data/pg_hba.conf'
+  tags: [db, postgresql]
+
+- name: yum | MD5-encrypted password for postgres 2
+  become: yes
+  command: 'sed -i "s/\(host.*all.*all.*::1\/128.*\)ident/\1md5/" /var/lib/pgsql/data/pg_hba.conf'
```

Comment:
Same, Ansibe module http://docs.ansible.com/ansible/lineinfile_module.html can help to make sure the command is Ansible-idempotent